source/_posts/Rails-Starter-Template.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Rails Starter Template
+date: 2020-10-05 19:18:31
+author: Anuj Middha
+category: Rails
+tags:
+- Ruby
```

Comment:
Use 1-2 long tail keyword

Reply:
Its not relevant, we cannot use non relevant keywords